test/examples/check-topics.example.ts

```diff
@@ -1,18 +1,7 @@
-import { describe, it, expect } from "vitest";
+import { describe, it, expect, vi } from "vitest";
 import * as ROSLIB from "../../src/RosLib.js";
 
-const expectedTopics = [
-  /*
-   * '/turtle1/cmd_vel', '/turtle1/color_sensor', '/turtle1/pose',
-   * '/turtle2/cmd_vel', '/turtle2/color_sensor', '/turtle2/pose',
-   */
-  "/tf2_web_republisher/status",
-  "/tf2_web_republisher/feedback",
-  // '/tf2_web_republisher/goal', '/tf2_web_republisher/result',
-  "/fibonacci/feedback",
-  "/fibonacci/status",
-  "/fibonacci/result",
-];
```

Comment:
This test data is ROS-1-only and wasn't even sensical.. it didn't check for _the one thing unique to our test suites_, the `/listener` topic 🤦🏻‍♂️